src/main/java/com/rocana/lucene/codec/v1/RocanaSearchCodecV1.java

```diff
@@ -54,4 +92,23 @@ Codec getWrappedCodec() {
     return delegate;
   }
 
+  /**
+   * This codec wraps the {@link Lucene54Codec}, which itself
+   * returns a {@link PerFieldPostingsFormat} that wraps
+   * {@link RocanaLucene50PostingsFormat}.
+   *
+   * We'll do the same thing, but instead we'll return
+   * our fork of {@link RocanaLucene50PostingsFormat}.
+   */
+  @Override
+  public PostingsFormat postingsFormat() {
+    return new PerFieldPostingsFormat() {
+
+      @Override
+      public PostingsFormat getPostingsFormatForField(String field) {
+        return forkedPostingsFormat;
+      }
+    };
+  }
```

Comment:
This implementation creates and returns a new `PerFieldPostingsFormat` for every invocation, whereas the Lucene code constructs a single object and always returns that.

This is the code in `Lucene54Codec`:
```
  @Override
  public final PostingsFormat postingsFormat() {
    return postingsFormat;
  }
```

If there is a DGR for doing this, that should be documented. Otherwise, let's follow the Lucene model and create fewer objects.